src/main/kotlin/com/coder_rangers/mobius_api/error/exceptions/NoFinishedTestException.kt

```diff
@@ -0,0 +1,3 @@
+package com.coder_rangers.mobius_api.error.exceptions
+
+class NoFinishedTestException(id: Long) : BadRequestException("The patient [$id] has no finished the test.")
```

Comment:
No sería **NOT**? El **No** solo me suena a cuando querés decir, por ejemplo, que no tenés ningún test terminado. 

Creo que sería más apropiado que se llame `TestNotFinishedException` y que el mensaje diga `The patient with id [$id] has not finished the test`. Traducido: `El paciente con id [$id] no ha terminado el test`

Reply:
me comi la t y la traduccion ahorratela porque se ingles